spec/amp-html-templates.md

```diff
@@ -50,13 +50,29 @@ submitted to the AMP repository.
 Templates can be defined anywhere in the AMP document's `body`, like this:
 
 ```html
+<!--
+Using the template tag
+
+The `type` attribute must reference the template's type as defined in the `custom-template` attribute when the templating system was imported in the document's `head`.
+-->
 <template type="amp-mustache">
   Hello {{world}}!
 </template>
 ```
+or like this:
+
+```html
+<!--
+Using the script tag
+
+The template type is defined in the `template` attribute.
```

Comment:
Mention that `<template>` is preferred but `<script>` can be helpful when working with tables or server-side rendering.